content/en/docs/concepts/sdk-configuration/exporter-selection-configuration.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Exporter Selection Configuration"
+description: >-
+ Environment Variables for configuring which exporter to use.
+weight: 10
+---
+
+The following environment variables let you configure one or more exporters when set.
+
+## `OTEL_TRACES_EXPORTER`
+
+Specifies which exporter is used for traces.
+
+**Default value:** `"otlp"`
+
+**Example:**
+
+`export OTEL_TRACES_EXPORTER="jaeger"`
+
+Accepted values for are:
+
+- `"otlp"`: [OTLP][spec-otlp]
+- `"jaeger"`: export in Jaeger data model
+- `"zipkin"`: [Zipkin](https://zipkin.io/zipkin-api/) (Defaults to [protobuf](https://github.com/openzipkin/zipkin-api/blob/master/zipkin.proto) format)
```

Comment:
Where have you found that it defaults to protobuf? E.g. the [.NET SDK](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.Zipkin)uses JSON serialization.

See also https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#zipkin-exporter

> This will be used to specify whether or not the exporter uses v1 or v2, json, thrift or protobuf. As of 1.0 of the specification, there is no specified default, or configuration via environment variables.